src/Analyzer/EventListener/ExpressionListener.php

```diff
@@ -15,11 +15,17 @@ class ExpressionListener extends EventListener
      */
     private $analyzers;
 
+    /**
+     * @param array $analyzers
+     */
     public function __construct(array $analyzers)
     {
         $this->analyzers = $analyzers;
     }
 
+    /**
+     * @param \PHPSA\Compiler\Event\ExpressionBeforeCompile $event
```

Comment:
I thought about something: do we really need this kind of comment/phpDoc? I don't really see the value in them as they give the exact same information as the type-hint below.

Open question: is it okay, in the future, to omit such redundant docblocks?

More on this in the "Minimal phpdoc" section of [this blog post](http://mnapoli.fr/approaching-coding-style-rationally/).